app/controllers/teams_controller.rb

```diff
@@ -133,7 +133,8 @@ def destroy
       flash[:notice] = I18n.t('teams.destroyed')
       redirect_to(set_destination(@team))
     else
-      render action: :show
+      flash[:alert] = I18n.t('teams.error')
+      redirect_to team_path(@team)
```

Comment:
I think this message is being displayed whatever the error is, not just active children